### ISSUE TITLE
Oracle 12.1.0.2-ee crashes upon start

### ISSUE THREAD

**Issue by Niceplace** [oracle/docker-images#302]
**Docker version** : 1.12.6
**Host** : RHEL 7.2 (Maipo)

I successfully built the 12.1.0.2-ee image with the buildDockerImage.sh script. I then try to run it with the following docker command :
```
docker run --name oracle12-ee \ 
-p 1521:1521 -p 5500:5500  \
-e ORACLE_SID=ORCLCDB \
-e ORACLE_PDB=ORCLPDB1 \
-e ORACLE_CHARACTERSET=AL32UTF8 \
 -v /app/docker-volumes/oracle/12:/opt/oracle/oradata \
 oracle/database:12.1.0.2-ee
```
And I get this output ...

```
/opt/oracle/runOracle.sh: line 89: [: -lt: unary operator expected
ORACLE AUTO GENERATED PASSWORD FOR SYS, SYSTEM AND PDBAMIN: uMmeHCOWWzE=1
UnsatisfiedLinkError exception loading native library: njni12
java.lang.UnsatisfiedLinkError: /opt/oracle/product/12.1.0.2/dbhome_1/lib/libnjni12.so: /opt/oracle/product/12.1.0.2/dbhome_1/lib/libclntsh.so.12.1: file too short
Exception in thread "main" java.lang.UnsatisfiedLinkError: oracle.net.common.NetGetEnv.get(Ljava/lang/String;)Ljava/lang/String;
        at oracle.net.common.NetGetEnv.get(Native Method)
        at oracle.net.config.Config.getNetDir(Unknown Source)
        at oracle.net.config.Config.initConfig(Unknown Source)
        at oracle.net.config.Config.<init>(Unknown Source)
        at oracle.sysman.assistants.util.NetworkUtils.<init>(NetworkUtils.java:298)
        at oracle.sysman.assistants.util.step.StepContext.<init>(StepContext.java:388)
        at oracle.sysman.assistants.dbca.backend.Host.<init>(Host.java:1050)
        at oracle.sysman.assistants.dbca.backend.Host.<init>(Host.java:1043)
        at oracle.sysman.assistants.dbca.backend.SilentHost.<init>(SilentHost.java:153)
        at oracle.sysman.assistants.dbca.Dbca.getHost(Dbca.java:198)
        at oracle.sysman.assistants.dbca.Dbca.execute(Dbca.java:121)
        at oracle.sysman.assistants.dbca.Dbca.main(Dbca.java:226)
cat: /opt/oracle/cfgtoollogs/dbca/ORCLCDB/ORCLCDB.log: No such file or directory
cat: /opt/oracle/cfgtoollogs/dbca/ORCLCDB.log: No such file or directory
sqlplus: error while loading shared libraries: /opt/oracle/product/12.1.0.2/dbhome_1/lib/libclntsh.so.12.1: file too short
mkdir: cannot create directory '/opt/oracle/oradata/dbconfig': Permission denied
mv: cannot stat '/opt/oracle/product/12.1.0.2/dbhome_1/dbs/spfileORCLCDB.ora': No such file or directory
mv: cannot stat '/opt/oracle/product/12.1.0.2/dbhome_1/dbs/orapwORCLCDB': No such file or directory
mv: cannot move '/opt/oracle/product/12.1.0.2/dbhome_1/network/admin/tnsnames.ora' to '/opt/oracle/oradata/dbconfig/ORCLCDB/': No such file or directory
cp: cannot create regular file '/opt/oracle/oradata/dbconfig/ORCLCDB/': No such file or directory
ln: failed to create symbolic link '/opt/oracle/product/12.1.0.2/dbhome_1/network/admin/tnsnames.ora': File exists
cp: cannot stat '/opt/oracle/oradata/dbconfig/ORCLCDB/oratab': No such file or directory
```

**Comment by gvenzl** [comment]
Hi @Niceplace, an `UnsatisfiedLinkError` would suggest that something did go wrong during the installation. Can you share the build log by any chance so that we can check what the installer log says.

**Comment by Niceplace** [comment]
Thanks for answering ! 

Unfortunately I couldn't paste the log in here, the file was 12 000 + lines long. I wouldn't have been very efficient anyways.

I did a bit more investigating on my part with the log file and it seems that GCC was missing, so I installed the "Development Tools" package from yum and these errors went away.

Now I have other problems but they are related to permissions and volumes, so I guess we can close this.


Thanks for the help anyways :) !
